src/app/Dashboard/DashboardLayoutToolbar.tsx

```diff
@@ -346,6 +351,9 @@ export const DashboardLayoutToolbar: React.FC<DashboardLayoutToolbarProps> = (_p
         <DropdownItem key="download" itemId={'download'}>
           {t('DashboardLayoutToolbar.DOWNLOAD_AS_TEMPLATE')}
         </DropdownItem>
+        <DropdownItem key="clearAll" itemId={'clearAll'}>
+          {t('DashboardLayoutToolbar.CLEAR_ALL_CARDS')}
```

Comment:
Lets call this key `CLEAR_LAYOUT`? You will need to run `yarn localize` again and update locale json :))

Reply:
@aali309 missed this.